addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "<%= htmlComponentName %>",
   "version": "0.0.0",
   "license": "Apache-2.0",
+  "scripts":{
+    "postinstall": "webdriver-manager update"
```

Comment:
Hm..... perhaps we can do something more involved in `ng test`. Like only running `webdriver-manager update` if there is no driver at all. 

I checked and `webdriver-manager` api doesn't seem to have something to do that. Perhaps I could do a PR to implement such functionality in protractor itself, a sort of `webdriver-manager initialize`.

@juliemr do you think this would be reasonable?